tests/backstop_data/engine_scripts/puppet/civicase/add.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = async (engine, scenario, vp) => {
+  const isWysiwygVisible = await engine.evaluate((selector) => {
+    const e = document.querySelector(selector);
+
+    if (!e) {
+      return false;
+    }
+
+    const style = window.getComputedStyle(e);
+
+    return style && style.display !== 'none' && style.visibility !== 'hidden' && style.opacity !== '0' && e.offsetHeight > 0;
+  }, '.crm-form-wysiwyg');
+
+  if (isWysiwygVisible) {
```

Comment:
i don't get what we are doing here. First you "manually" check if the wysiwyg is visible, then you tell puppeteer to wait until said wyiswyg is visible again? Why doesn't using `waitFor(, { visible: true })` directly work?

Reply:
Sorry for the confusion. Fixed the code.